test/passport_test.coffee

```diff
@@ -53,6 +66,89 @@ describe "Register / Login", ->
                 body.isAuthenticated.should.be.ok
                 done()
 
+    describe "Login with two-factors authentication: TOTP", ->
+        
+        before (done) ->
+            # Logging out
+            client.get "logout", (error, response, body) ->
+                # Adding TOTP setting to the user's settings
+                user.request "all", (error, users) ->
+                    user.updateAttributes users[0]._id,
```

Comment:
According to [the documentation](https://cozy.github.io/cozy-db/doc/DOCINDEX.md.html#toc_2), yes it does (and I could confirm it during my tests).